index.html

```diff
@@ -66,14 +66,191 @@ <h1>I’m Tracy<br />Glad to see,You!</h1>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="grid-container2">
+      <div class="about-me">
+        <h1>
+          About<br />
+          Myself
+        </h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <a href="#" style="margin-left: 2px">LET’S CONNECT</a>
+        <div class="lnks">
+          <ul>
+            <li>
+              <a href="#"
+                ><i class="fa-brands fa-github" style="margin-left: -70px; color: #505F79; font-weight: 800;font-size: 22px;"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i
+                  class="fa-brands fa-linkedin-in"
+                  style="margin-left: -45px; color: #505F79; font-weight: 800; font-size: 22px;"
+                ></i
+              ></a>
+            </li>
+             <li>
+              <a href="#"><img src="./images/Vector (1).svg" style="margin-left: -25px; color: #505F79;font-weight: 800; width: 17px;"/></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fa-brands fa-twitter" style="margin-left: -15px; color: #505F79; font-weight: 800; font-size: 22px;"></i></a>
+            </li>
+            
+            <li>
+              <a href="#"
+                ><i class="fa-solid fa-m" style="margin-left: -5px; color: #505F79;font-weight: 800; font-size: 22px;"></i
+              ></a>
+            </li>
+          </ul>
+          <button><a href="#">Get my resume</a></button>
+        </div>
+      </div>
+      <div class="skills">
+        <ul class="language">
+          <li>
+            <a class="lang" href="#"
+              >Languages
+              <img class="arrow" src="./images/drop-down-arrow-4.png"
+            /></a>
+
+            <ul class="sub-lang">
+              <li>
+                <img src="./images/Ellipse 1.png" />
+                <a class="jav" href="#">JavaSript</a>
+              </li>
+              <li>
+                <img src="./images/Ellipse2.png" />
+                <a href="#">HTML</a>
+              </li>
+              <li>
+                <img src="./images/Ellipse3.png" />
+                <a href="#">CSS</a>
+              </li>
+            </ul>
+          </li>
```

Comment:
- [ ] Good job using the required tags on this section of you page :clap: , however , make sure the padding of the `li` sticks to the design given. 
- [ ] Also make sure that the `ul` is correctly centered as in the Figma design, I hope the picture below speaks well that my text :smile: 

Your design
![Screenshot from 2023-01-13 15-05-57](https://user-images.githubusercontent.com/80458199/212327860-7e545f83-2209-46c4-8fdf-fdfa22cce434.png)

Figma Design
![Screenshot from 2023-01-13 15-06-19](https://user-images.githubusercontent.com/80458199/212327897-296782fe-152b-435d-9c12-bd448af419bc.png)